src/orquestra/sdk/_base/cli/_dorq/_workflow/_submit.py

```diff
@@ -54,13 +58,7 @@ def _on_cmd_call_with_exceptions(
         Implementation of the command action. Doesn't catch exceptions.
         """
         # 1. Resolve config
-        # TODO: prompt user for selecting config.
-        # Jira ticket: https://zapatacomputing.atlassian.net/browse/ORQSDK-674
-        if config is None:
-            raise ValueError(
-                "Temporarily, 'config' needs to be passed explicitly. Sorry!"
-            )
-        resolved_config = config
+        resolved_config = self._config_resolver.resolve(None, config)
```

Comment:
**non-blocking**: the previously implemented config resolver was intended for use with commands that require both `wf-run-id` and `config`. Submit doesn't use `wf-run-id` (this value is generated as a result of submission). Using `self._config_resolver.resolve(None, config)` _accidentally_ works because of the fact that we pass through the `config` value even if we don't pass wf run ID to `orq wf view`, which an implementation detail of the resolver for other commands and might easily break. 

WDYT about making a separate arg resolver just for `orq wf submit`? Internally it can wrap the current `ConfigResolver` but we would have a clear place where we make the assumption about internal workings of the resolution.